diagnostic.md

```diff
@@ -6,37 +6,38 @@ Write your answers inside this file, where it's indicated by the comments.
 What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
-```sh
-<!-- Remove this comment and place your answer here. -->
+```sh copy the link git@github.com:skylarkJ/git-basics-diagnostic.git
+git clone git@github.com:skylarkJ/git-basics-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+cd .. that repository from fork
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git checkout -b response - will create breanch and switch to it
 ```
 
 4.We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git status
 ```
 
 5.We want to prepare a change for a commit by adding a file to the staging
     area. What command do we use to stage a file named `diagnostic.md`?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git add diagnostic.md
 ```
 
 6.Once `diagnostic.md` is staged, we have to make a commit by `git commit`.
 What are the two formatting items you **need** to make up your commit message?
 
 <!-- Remove this comment and place your answer here. -->
+have the action as a verb in order - do, finish
```

Comment:
A good commit message consists of a short summary title and a body detailing the changes made.